docs/troubleshooting.rst

```diff
@@ -18,15 +18,44 @@ These commands will reset your environment to a "fresh" version, while
 retaining the database::
 
   cd /path/to/kuma
-  docker-compose stop
-  docker-compose rm
+  docker-compose down
   make clean
   git submodule sync --recursive && git submodule update --init --recursive
   docker-compose pull
   docker-compose build --pull
   docker-compose up
 
 
+Reset a corrupt database
+------------------------
+The Kuma database can become corrupted if the system runs out of disk space,
+or is unexpectedly shutdown. MySQL can repair some issues, but sometimes you
+have to start from scratch. When this happens, pass an extra argument
+``--volumes`` to ``down``::
+
+  cd /path/to/kuma
+  docker-compose down --volumes
+  make clean
+  git submodule sync --recursive && git submodule update --init --recursive
+  docker-compose pull
+  docker-compose build --pull
+  docker-compose up -d mysql
```

Comment:
So in local development I think its not needed to start the ``mysql`` container before.